test/notifications_test.js

```diff
@@ -121,14 +145,18 @@ describe('notifications', function() {
   describe('once subscriptions are registered', function() {
     beforeEach(function*() {
       mq.connect(mqSocket);
-      yield* api.createSubscription(subscription);
+      for (let i = 0; i < users.length; i++) {
+        yield* api.login(users[i].email, users[i].password);
+        yield* api.createSubscription(subscriptions[i]);
+      }
     });
 
     afterEach(function() {
       mq.disconnect(mqSocket);
     });
 
-    it('does not send a reminder twice to the message queue', function*() {
+    it('sends reminders to the right person and ' +
+       'does not send a reminder twice to the message queue', function*() {
```

Comment:
this test was actually preexisting; I just changed the setup to have several users and reminders for each one of these users.